pom.xml

```diff
@@ -363,37 +363,6 @@
                     <failOnViolation>true</failOnViolation>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
?

Reply:
sorry iterating on this just testing some things out with travis might have snuck in